suite/tests/api/ir_aarch64.c

```diff
@@ -2480,6 +2480,142 @@ test_asimdsame(void *dc)
     test_instr_encoding(dc, OP_bif, instr);
 }
 
+static void
+test_asimd_mem(void *dc)
+{
+    byte *pc;
+    instr_t *instr;
+
+    /* Advanced SIMD memory (multiple structures) */
+
+    /* Load multiple 1-element structures (to 1, 2, 3 or 4 registers)
+       Naming convention based on official mnemonics:
+       INSTR_CREATE_ld1_multi_<n>() where <n> is 1, 2, 3 or 4
+
+       LD1 { <Vt>.<T> }, [<Xn|SP>]
+       LD1 { <Vt>.<T>, <Vt2>.<T> }, [<Xn|SP>]
```

Comment:
Currently, only the 1 operand version is tested, right?

Reply:
Yes. Hopefully more will follow as time allows.

Reply:
Hmm...good spot. Perhaps I should remove those comments as they could be misleading?

Reply:
Maybe not. The header comments are useful placeholders and each test call is explicitly commented to make it clear it's the 1 operand version being run.